tests/demo/pio_drv_simple.h

```diff
@@ -0,0 +1,7 @@
+#include <stdlib.h>
+#include <stdint.h>
+
+#define PIO_BASE 0x60000
```

Comment:
You shouldn't hardcode this address, since this will not be true for every usage of the PIO block in a larger system. In fact, this will prevent handling the case where there are two separate PIO instances with two separate addresses.

Shouldn't this info come from the `struct metal_pio` that is passed in to each driver function?

Reply:
I think I was following the example drivers too closely. will redo.

Reply:
> It's still kind of hardcoded, until we get decent OM/dts integration

In a separate PR, could you update the PIO block's Chisel wrapper using the latest `duh-scala` so that the DTS node gets created. We don't even need any custom properties yet, since just getting the standard properties (`compatible` and `reg`) would enable properly getting this information.

Reply:
will do